app/decorators/models/solidus_bolt/shipment_decorator.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module SolidusBolt
+  module ShipmentDecorator
+    def self.prepended(base)
+      base.class_eval do
+        before_save :update_bolt_tracking_info
+      end
+    end
+
+    def update_bolt_tracking_info
+      return unless tracking_was.nil?
```

Comment:
I think that `return if tracking_was` is more readable

Reply:
Modified here [4644cca](https://github.com/nebulab/solidus_bolt/commit/4644ccac158814c90754f895bcb8cb7aace3a750)